docs/Learning-Environment-Create-New.md

```diff
@@ -473,7 +420,7 @@ localPosition rather than position, so that our position reference is in referen
 to the prefab TrainingArea's location, and not global coordinates.
 
 1. Replace all references of `this.transform.position` in RollerAgent.cs with `this.transform.localPosition`.
-2. Replace all references of `Target.position` in RollerAgent.cs with `Target.localPosition`.
+1. Replace all references of `Target.position` in RollerAgent.cs with `Target.localPosition`.
```

Comment:
Should we just start with these in the example, instead of changing them later? There's been some confusion before because the examples conflict with the best practices that we call out elsewhere.

Reply:
Makes sense.